azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -187,8 +184,8 @@ public void testPreKilledRun() {
     Assert.assertTrue(runner.getStatus() == node.getStatus());
     Assert.assertTrue(node.getStatus() == Status.KILLED);
     Assert.assertTrue(node.getStartTime() > 0 && node.getEndTime() > 0);
-    // Give it 10 ms to fail.
-    Assert.assertTrue(node.getEndTime() - node.getStartTime() < 10);
+    // Give it 2000 ms to fail.
+    Assert.assertTrue(node.getEndTime() - node.getStartTime() < 2000);
```

Comment:
This change is in already enabled test `testPreKilledRun`: relying on time constrains as short as 10 ms is too error prone. I think this test can serve its purpose by asserting the status (KILLED) without enforcing this kind of temporal check too tightly.